### PR TITLE
LUT-27422 : add elements to create slot entries

### PR DESCRIPTION
**Pull request by briceboutamdja**
add elements to create slot entries

### CODE REVIEW

**Comment by FrancoisEricMerlin** [comment]
hello can you resolved clonflicts please

File: src/java/fr/paris/lutece/plugins/forms/service/entrytype/EntryTypeSlot.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	/**
+     * Name of the bean of this service
+     */
+    public static final String BEAN_NAME = "forms.entryTypeSlot";
+    
+	private static final String TEMPLATE_CREATE = "admin/plugins/forms/entries/create_entry_type_slot.html";
+	private static final String TEMPLATE_MODIFY = "admin/plugins/forms/entries/modify_entry_type_slot.html";
+	private static final String TEMPLATE_READONLY_BACKOFFICE = "admin/plugins/forms/entries/readonly_entry_type_slot.html";
+	private static final String TEMPLATE_EDITION_BACKOFFICE = "admin/plugins/forms/entries/fill_entry_type_slot.html";
+    private static final String TEMPLATE_EDITION_FRONTOFFICE = "skin/plugins/forms/entries/fill_entry_type_slot.html";
```
Comment (FrancoisEricMerlin) [inline src/java/fr/paris/lutece/plugins/forms/service/entrytype/EntryTypeSlot.java:19]: format your code

Comment (briceboutamdja) [reply src/java/fr/paris/lutece/plugins/forms/service/entrytype/EntryTypeSlot.java]: done

File: src/java/fr/paris/lutece/plugins/forms/service/entrytype/AbstractEntryTypeSlot.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+import java.util.Locale;
+
+import static fr.paris.lutece.plugins.forms.util.FormsConstants.MESSAGE_ERROR_CRENEAUX;
+import static fr.paris.lutece.plugins.forms.util.FormsConstants.MESSAGE_ERROR_CRENEAUX_IMPOSSIBLE;
+
+public abstract class AbstractEntryTypeSlot extends EntryTypeService
+{
+    // PARAMETERS
+    public static final String PARAMETER_SUFFIX_BEGIN       = "_debut";
+    public static final String PARAMETER_SUFFIX_END         = "_fin";
```
[the remark this answers is not in the document]
Comment (briceboutamdja) [reply src/java/fr/paris/lutece/plugins/forms/service/entrytype/AbstractEntryTypeSlot.java:22]: done

File: src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties [hunk 3]
```diff
@@ -1010,6 +1010,8 @@ xpage.myformresponses.view.labelView=View
 message.error.noStep=No content available
 message.view.loadBackUp=Load from backup: {0}
 message.error.control={0}
+message.error.creneaux=Slots must not be unfilled
```
[the remark this answers is not in the document]
Comment (briceboutamdja) [reply src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties]: done

File: src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties [hunk 4]
```diff
@@ -385,6 +385,9 @@ manage_question_response.step=Step
 manage_question_response.status=Published
 manage_question_response.order=Order
 
+message.slot.determinant=From
```
Comment (seboo) [inline src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties:388]: All property keys must be in english, please check all the keys

File: src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java [hunk 5]
```diff
@@ -189,6 +189,8 @@ public final class FormsConstants
     public static final String MESSAGE_LOAD_BACKUP = "forms.message.view.loadBackUp";
     public static final String MESSAGE_SUMMARY_TITLE = "forms.summary.title";
     public static final String MESSAGE_ERROR_NO_STEP = "forms.error.form.noStep";
+    public static final String MESSAGE_ERROR_CRENEAUX = "forms.message.error.creneaux";
```
Comment (seboo) [inline src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java:192]: all constants must be in english language,
please check all constants

**Review by seboo — CHANGES_REQUESTED** [review]

File: src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties (hunk 4 above)
Comment (briceboutamdja) [reply src/java/fr/paris/lutece/plugins/forms/resources/forms_messages.properties]: sorry, I used a translator though as I wasn't sure how to qualify the nature of the word. and it's done in the hope that this translation will go down well.

File: src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java (hunk 5 above)
Comment (briceboutamdja) [reply src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java]: deleted because not used in addition to present French

**Pull request merged**